public/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
I don't think we need this file it looks like it's for vanillaJS apps. You also haven't included a reference for vite to use this file